e2etests/web/text_editing/.flutter-plugins

```diff
@@ -0,0 +1,2 @@
+# This is a generated file; do not edit or check into version control.
```

Comment:
Sounds like you need a `.gitignore` file.

Reply:
done.